03/pwilson802/tags.py

```diff
@@ -0,0 +1,59 @@
+from collections import Counter
+from difflib import SequenceMatcher
+from itertools import product
+import re
+
+IDENTICAL = 1.0
+TOP_NUMBER = 10
+RSS_FEED = 'rss.xml'
+SIMILAR = 0.87
+TAG_HTML = re.compile(r'<category>([^<]+)</category>')
+
+
+def get_tags():
+    """Find all tags (TAG_HTML) in RSS_FEED."""
+    tags = []
+    with open(RSS_FEED) as file:
+        for line in file.readlines():
+            for tag in TAG_HTML.findall(line):
+                tags.append(tag.replace('-', ' ').lower())
+    return tags
+
+def get_top_tags(tags):
+    """Get the TOP_NUMBER of most common tags"""
+    tag_count = Counter(tags)
+    return tag_count.most_common(10)
+
```

Comment:
run flake8, it would complain about 1 and 3 lines between functions (should be 2)